wpilibcExamples/src/main/cpp/templates/robotbaseskeleton/cpp/Robot.cpp

```diff
@@ -20,7 +21,15 @@ void Robot::Teleop() {}
 void Robot::Test() {}
 
 void Robot::StartCompetition() {
-  frc::internal::DriverStationModeThread modeThread;
+  HAL_ControlWord word;
+  HAL_GetControlWord(&word);
```

Comment:
Should we update this example to use `frc::DriverStation::GetControlWord()` instead?